Model/AdyenInitiateTerminalApi.php

```diff
@@ -72,6 +73,7 @@ public function __construct(
         $client = new \Adyen\Client();
         $client->setApplicationName("Magento 2 plugin");
         $client->setXApiKey($apiKey);
+        $client->setInputType('json');
```

Comment:
shouldn't we better implement http://php.net/manual/en/function.is-string.php or http://php.net/manual/en/function.is-array.php on lib instead of explicitly asking the inputType?

Reply:
@lancergr agree if you can do a quick PR on the PHP library I would say just do it if you think it cost more time create an issue on youtrack to refine.